roadmap/implementers-guide/src/node/backing/statement-distribution.md

```diff
@@ -1,107 +1,453 @@
+-   We only consider statements as "importable" when the corresponding candidate
+    is part of the hypothetical frontier, and only send "importable" statements to
+    the backing subsystem itself.
+
+### Cluster Mode
+
+-   Validator nodes are partitioned into groups (with some exceptions), and
+    validators within a group at a relay-parent can send each other `Statement`
+    messages for any candidates within that group and based on that relay-parent.
+    This is referred to as the "cluster" mode.
+-   `Seconded` statements must be sent before `Valid` statements.
+-   `Seconded` statements may only be sent to other members of the group when the
+    candidate is fully known by the validator.
+-   Validators in a cluster receiving messages about unknown candidates request
+    the candidate (and statements) from other cluster members which have it.
+-   Spam considerations
+    -   The maximum depth of candidates allowed in asynchronous backing determines the
+        maximum amount of `Seconded` statements originating from a validator V which
+        each validator in a cluster may send to others. This bounds the number of
+        candidates.
+    -   There is a small number of validators in each group, which further limits
+        the amount of candidates.
+
+&#x2014;
+
+-   We accept candidates which don't fit in the fragment trees of any relay
+    parents.
+    -   We listen to prospective parachains subsystem to learn of new additions to
+        the fragment trees.
+    -   Use this to attempt to import the candidate later.
+
+### Grid Mode
+
+-   Every consensus session provides randomness and a fixed validator set, which
+    is used to build a redundant grid topology.
+-   This grid topology is used to create "sending" and "receiving" paths from each
+    validator group to every validator.
+-   When a node observes a candidate as backed, it sends a
+    `BackedCandidateManifest` to their "receiving" nodes.
+-   If receiving nodes don't yet know the candidate, they request it.
+-   Once they know the candidate, they respond with a
+    `BackedCandidateAcknowledgement`.
+-   Once two nodes perform a manifest/acknowledgement exchange, they can send
+    `Statement` messages directly to each other for any new statements they might
+    need.
+-   There are limitations on the number of candidates that can be advertised by
+    each peer, similar to those in the cluster. Validators do not request
+    candidates which exceed these limitations.
+-   Validators request candidates as soon as they are advertised, but do not
+    import the statements until the candidate is part of the hypothetical
+    frontier, and do not re-advertise or acknowledge until the candidate is
+    considered both backed and part of the hypothetical frontier.
+-   Note that requesting is not an implicit acknowledgement, and an explicit
+    acknowledgement must be sent upon receipt.
+
+### Old Grid Protocol
```

Comment:
The old grid protocol here has some information that the new one leaves out. I'm wanting to merge it into the new one, but wondering how much of this is still relevant?

Reply:
Since this version of the protocol is more detailed, I can move it to the module doc.

Reply:
FYI @rphmeier I removed this section from here, and pushed it to 5999 as a module doc on `grid.rs`. I figured it had already been reviewed and didn't need its own PR.